src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.exe.manifest

```diff
@@ -1,18 +1,38 @@
-<?xml version="1.0" encoding="utf-8"?>
-<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-  <assemblyIdentity version="1.0.0.0" name="CppTestApp"/>
-  <file name="WinRT.Host.dll">
-    <activatableClass
-        name="AuthoringSample.BasicClass"
-        threadingModel="both"
-        xmlns="urn:schemas-microsoft-com:winrt.v1" />
-    <activatableClass
-        name="AuthoringSample.CustomWWW"
-        threadingModel="both"
-        xmlns="urn:schemas-microsoft-com:winrt.v1" />
-    <activatableClass
-        name="AuthoringSample.TestClass"
-        threadingModel="both"
-        xmlns="urn:schemas-microsoft-com:winrt.v1" />
-  </file>
+<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="CppTestApp"/>
+  <file name="WinRT.Host.dll">
+    <activatableClass
+        name="AuthoringSample.BasicClass"
+        threadingModel="both"
+        xmlns="urn:schemas-microsoft-com:winrt.v1" />
+    <activatableClass
+        name="AuthoringSample.CustomDictionary"

```

Comment:
We have to do this for every public class/type they author right? Not for this PR, but maybe in the future we can add something that writes this manifest file based on the authored class

Reply:
I agree that we should look at automating this later on when we add more project support.